src/boot.lisp

```diff
@@ -448,20 +448,30 @@
 
 (defparameter *features* '(:jscl :common-lisp))
 
+;;; @vlad-km 20112022
```

Comment:
Maybe we can leave the dates out, it's easily accessible  with git blame anyway :-)

Reply:
see below